skin-deep.js

```diff
@@ -156,6 +156,10 @@ function findNodes(node, predicate, findOne) {
 }
 
 function getTextFromNode(node) {
+  // Ignore null/undefined children
+  if (node === null || node === undefined) {
```

Comment:
False is technically an object so it doesn't generate the error when properties are checked against :)